### PR TITLE
Update Slovenian translation

### PR DESCRIPTION
**Pull request by mojca**
Not quite complete, but slowly getting somewhere.

### CODE REVIEW

File: trans/TeXworks_sl.ts [hunk 1]
```diff
@@ -879,7 +879,7 @@ Do you want to continue?</source>
         <location filename="../src/PrefsDialog.cpp" line="543"/>
         <source> (%1)</source>
         <extracomment>Country suffix for TeXworks translations (ex. &quot;Portuguese (Brazil)&quot;)</extracomment>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished"> (%1)</translation>
```
Comment (mojca) [inline trans/TeXworks_sl.ts]: I didn't quite understand what you want here / why is this even translatable.

File: trans/TeXworks_sl.ts [hunk 2]
```diff
@@ -1103,35 +1103,35 @@ Do you want to continue?</source>
...
     <message>
         <location filename="../src/PrefsDialog.ui" line="765"/>
         <location filename="../src/PrefsDialog.ui" line="797"/>
         <source>Highlight Line</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Označi vrstico</translation>
     </message>
     <message>
         <location filename="../src/PrefsDialog.ui" line="773"/>
         <source>Preview &gt; Source :</source>
```
Comment (mojca) [inline trans/TeXworks_sl.ts]: Is this inconsistent spacing in the original (why space before the color)?

File: trans/TeXworks_sl.ts (hunk 1 above)
Comment (stloeffler) [reply trans/TeXworks_sl.ts]: You are right, this should be changed

File: trans/TeXworks_sl.ts (hunk 2 above)
Comment (stloeffler) [reply trans/TeXworks_sl.ts]: Yep, that's a typo, there should not be a space before ":". Again, I'll change it after the release.

**Comment by stloeffler** [comment]
Thanks a lot - merged.

**Pull request merged**